00-spatio-temporal-science-questions.Rmd

```diff
@@ -225,214 +187,209 @@ achieve this goal we need data that:
+sensing imagery and products like NDVI. In the United States, one popular data
+provider is the United States Geological Survey.
 
 * <a href="http://phenology.cr.usgs.gov/get_data_main.php" target="_blank">Remote Sensing Phenology</a>
 * <a href="http://glovis.usgs.gov/" target="_blank">USGS GLOVIS</a>
 * <a href="http://landsatlook.usgs.gov/" target="_blank">Landsat Look Viewer</a>
-* <a href="http://landsat.gsfc.nasa.gov/" target="_blank">NASA Landsat</a>
 
-## Landsat Look Viewer
+Other, non-USGS sources of remote sensing data can be found in the 
+[Data Sources list at the end of this tutorial]({{ site.baseurl}}/R/spatio-temporal-research-questions#resources-for-locating-spatial-and-ecological-data).
 
 <div id="challenge" markdown="1">
+## Challenge: Explore *Landsat Look* Website
 
-## Challenge: Explore Landsat Look Website
-
-Visit the <a href="http://landsatlook.usgs.gov/" target="_blank">Landsat Look Viewer
-website</a>. Search for an location - for example `Harvard Forest, Massachusetts, 
-United States`. 
+Visit the 
+<a href="http://landsatlook.usgs.gov/" target="_blank">Landsat Look Viewer website</a>.
+Search for any location of interest. Can decide? Try one of our study sites: 
+Harvard Forest, Massachusetts, United States. 
 
+1. Data from which Landsat sensors are available for your area? 
+2. How is the quality of the images?
 </div>
 
 ```{r explore-landsat-data, echo=FALSE}
-#.
+# Answers for Harvard Forest; will vary by location
+# 1. Landsat 8 (good quality), Landsat 7 EMT+ SLC-off (some problem areas), 
+# Landsat 7 EMT+ SLC-on (one strip of missing data), Landsat 4-5 (good quality),
+# Landsat 1-5 (missing large amounts of data). 
 
 ```
 
-### Raster Data Format - Work With Remote Sensing Imagery 
-
-Remote sensing images are in **raster** format. They are composed of pixels - just like
-the pixels in the images we take with our camera phones. To work with raster data 
-in R, we can use the <a href="https://cran.r-project.org/web/packages/raster/index.html" target="_blank">`raster` package. </a>
-
-Because we are studying phenophase timing, we need raster data collected 
-repeatedly through time to identify when groups of plants green-up and brown down.
-We can use the <a href="https://cran.r-project.org/web/packages/rasterVis/index.html" target="_blank"> `rasterVis` package in `R` </a> efficiently plot raster time
-series.
+### Work With Remote Sensing Imagery - Raster Data Format 
 
-NOTE: We will cover working with raster data in R, in the [raster time series 
-lessons]({{ site.baseurl }}/tutorial-series/raster-data-series/)
-
-<figure>
-	<a href="https://xxxx">
-	<img src="http://data-lessons.github.io/NEON-R-Spatial-Raster//images/dc-spatial-raster/raster_concept.png"></a>
-	<figcaption> CHANGE URL TO DATA SKILLS
-	Source: National Ecological Observatory Network (NEON)  
-	</figcaption>
-</figure>
+Remote sensing images are in **raster** format. They are composed of pixels -
```

Comment:
See note above.  This isn't the time/lesson to introduce the full concept of Raster data.